pkg/model/awsmodel/autoscalinggroup.go

```diff
@@ -101,6 +101,9 @@ func (b *AutoscalingGroupModelBuilder) buildLaunchTemplateTask(c *fi.ModelBuilde
 	if err != nil {
 		return nil, fmt.Errorf("error building cloud tags: %v", err)
 	}
+	for k, v := range b.CloudTags(b.AutoscalingGroupName(ig), false) {
```

Comment:
So why don't we put these on the ASG? Should we put this call inside `CloudTagsForInstanceGroup` so they also get applied when using launch configurations?

Reply:
Sounds good to me. Will take a look to see where and how this is used.